packages/styles/components/_c.notification.scss

```diff
@@ -0,0 +1,109 @@
+$text-font-size: get-token(size, font, '04');
+$title-font-size: get-token(size, font, '05');
+
+.mc-notification {
+  box-sizing: border-box;
+  position: relative;
+  display: block;
+  border-width: get-border('s');
+  border-style: solid;
+  font-size: $text-font-size;
+  width: 288px;
```

Comment:
It's better if you don't assign a  a fixed pixel width to your component. You can take in example of the "input component", its width is 100%.
Components must be adaptative, because we can't know their futur context.

Reply:
Yes, It was a misunderstanding of mine, I thought Notification had width, but It hasn't at all.